app/models/concerns/aliased_index.rb

```diff
@@ -9,7 +9,7 @@ module AliasedIndex
 
   module ClassMethods
     def timestamped_index_name
-      [base_name, Time.current.to_s(:number)].join('-')
+      [base_name, Time.current.strftime('%Y%m%d%H%M%S%L')].join('-')
```

Comment:
`Time.current.to_s(:number)` still works on rails 7. I don't think we need to upgrade anything in here